.travis.yml

```diff
@@ -0,0 +1,15 @@
+dist: trusty
```

Comment:
I leave that choice to you, as you're surely more qualified to decide on this.

Personally, I always use Travis for code validation, to avoid bumping into the free GH Actions limits (since validation is done for every commit on every branch, having many repositories could quickly reach the limit).

Another reason to keep it on Travis (instead of Circle or GH Actions) might be performance, since it would run in a parallel job.

Reply:
I'd like to avoid using multiple CI providers; if we can do it all in GH actions that seems better to me.

Reply:
There's a ready availabe EClint action:

https://github.com/marketplace/actions/eclint

I haven't use GH Actions before, some I'm not sure how to set it up. Can someone else add the action?